AdyenDropIn/Views/ModalToolbar.swift

```diff
@@ -11,7 +11,7 @@ internal final class ModalToolbar: UIView, AnyNavigationBar {
     private let style: NavigationStyle
     internal var onCancelHandler: (() -> Void)?
     private let title: String?
-    private let paddingWithMarginCorrection: CGFloat = 20
+    private let paddingWithMarginCorrection: CGFloat = 16
```

Comment:
why this change?

Reply:
Arjen requested a fix on the modal view title margin so it could be aligned with the form's fields